### DISCUSSION TITLE
Error Finishing Installation

### DISCUSSION THREAD

**Discussion by cast-play** [AzuraCast/AzuraCast#7028] [General]
Hello! When installing Azuracast on Ubuntu 22.04, this error message appears at the end (below). The panel opens normally to configure by IP and it works, but I'm worried if something was affected during the installation and it might not work correctly. I'm a beginner, how can I solve it?

AzuraCast.ERROR: Doctrine\DBAL\Exception\DriverException: An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist (uncaught exception) at /var/azuracast/www/vendor /doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 90 while running console command `azuracast:setup` {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 1305): An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/API/MySQL/ ExceptionConverter.php:90)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1305): SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 42000): SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:27)"} []
AzuraCast.WARNING: Console command `azuracast:setup` exited with error code 1305. [] []

In ExceptionConverter.php line 90:

  An exception occurred while executing a query: SQLSTATE[42000]: Syntax error or access violation: 1305
  SAVEPOINT DOCTRINE_3 does not exist

In Exception.php line 28:

  SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist

In Connection.php line 27:

  SQLSTATE[42000]: Syntax error or access violation: 1305 SAVEPOINT DOCTRINE_3 does not exist

**Comment by BusterNeece** [comment]
This should be fixed as of the latest Rolling Release.
